docs/content/preview/yugabyte-platform/troubleshoot/universe-issues.md

```diff
@@ -401,7 +401,11 @@ You can create a support bundle as follows:
 
   The **Support Bundles** dialog allows you to either download the bundle or delete it if it is no longer needed. By default, bundles expire after ten days to free up space.
 
-## Collect core dumps in Kubernetes environments
+## Debug crashing YugabyteDB pods in Kubernetes
+
+If the YugabyteDB pods of your universe are crashing, you can debug them with the help of following instructions.
+
+### Collect core dumps in Kubernetes environments
 
 When dealing with Kubernetes-based installations of YugabyteDB Anywhere, you might need to retrieve core dump files in case of a crash within the Kubernetes pod. For more information, see [Specify ulimit and remember the location of core dumps](../../install-yugabyte-platform/prerequisites#specify-ulimit-and-remember-the-location-of-core-dumps).
```

Comment:
links to wrong location

Reply:
Yes, I fixed it my other PR, but let me fix here as we will merge this one first.